app/src/main/java/org/oppia/android/app/home/promotedlist/PromotedStoryListView.kt

```diff
@@ -32,6 +32,12 @@ class PromotedStoryListView @JvmOverloads constructor(
   @Inject
   lateinit var oppiaLogger: OppiaLogger
 
+  @Inject
+  lateinit var fragment: Fragment
```

Comment:
Ditto--is this needed?

Reply:
Not needed as of now, refactored out.

Reply:
It seems seems to be present--did you mean to remove it?